<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>900</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1100</width>
    <height>900</height>
   </size>
  </property>
  <property name="windowTitle">
   <string notr="true">NextBlock</string>
  </property>
  <property name="windowIcon">
   <iconset resource="nextblock.qrc">
    <normaloff>:/icons/nextblock.png</normaloff>:/icons/nextblock.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_0">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_1"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string/>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QGroupBox" name="groupBox_2">
               <property name="title">
                <string>Next block preview</string>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_6">
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout_5"/>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <widget class="QLabel" name="startSpeed">
                 <property name="text">
                  <string>Start speed</string>
                 </property>
                </widget>
               </item>
               <item alignment="Qt::AlignmentFlag::AlignLeft">
                <widget class="QSpinBox" name="spinBoxStartSpeed">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimum">
                  <number>1</number>
                 </property>
                 <property name="maximum">
                  <number>9</number>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer">
                 <property name="orientation">
                  <enum>Qt::Orientation::Horizontal</enum>
                 </property>
                 <property name="sizeType">
                  <enum>QSizePolicy::Policy::Minimum</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QCheckBox" name="checkBoxPreview">
                 <property name="focusPolicy">
                  <enum>Qt::FocusPolicy::NoFocus</enum>
                 </property>
                 <property name="text">
                  <string>Show next block</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="checkBoxRotate">
                 <property name="focusPolicy">
                  <enum>Qt::FocusPolicy::NoFocus</enum>
                 </property>
                 <property name="text">
                  <string>Rotate clockwise</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="checkBoxRotateStart">
                 <property name="focusPolicy">
                  <enum>Qt::FocusPolicy::NoFocus</enum>
                 </property>
                 <property name="text">
                  <string>Rotate on start</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="checkBoxSounds">
                 <property name="focusPolicy">
                  <enum>Qt::FocusPolicy::NoFocus</enum>
                 </property>
                 <property name="text">
                  <string>Sounds</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QGroupBox" name="groupBox_3">
               <property name="title">
                <string/>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout">
                <item>
                 <widget class="QLabel" name="labelScore">
                  <property name="font">
                   <font>
                    <pointsize>14</pointsize>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">color: rgb(0, 0, 255);</string>
                  </property>
                  <property name="text">
                   <string>Score: </string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="labelDeltaScore">
                  <property name="font">
                   <font>
                    <pointsize>14</pointsize>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">color: rgb(85, 85, 255);</string>
                  </property>
                  <property name="text">
                   <string>Score from last block: </string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="labelScorePerBlock">
                  <property name="font">
                   <font>
                    <pointsize>14</pointsize>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">color: rgb(170, 85, 255);</string>
                  </property>
                  <property name="text">
                   <string>Average score per block: </string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="labelSpeed">
                  <property name="font">
                   <font>
                    <pointsize>14</pointsize>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">color: rgb(255, 85, 0);</string>
                  </property>
                  <property name="text">
                   <string>Speed: </string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="labelBlocks">
                  <property name="font">
                   <font>
                    <pointsize>14</pointsize>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">color: rgb(170, 0, 0);</string>
                  </property>
                  <property name="text">
                   <string>Blocks:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="labelDelLines">
                  <property name="font">
                   <font>
                    <pointsize>14</pointsize>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">color: rgb(0, 85, 0);</string>
                  </property>
                  <property name="text">
                   <string>Deleted lines:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="labelEfficiency">
                  <property name="font">
                   <font>
                    <pointsize>14</pointsize>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">color: rgb(0, 170, 0);</string>
                  </property>
                  <property name="text">
                   <string>Efficiency: </string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_1">
             <item>
              <widget class="QGroupBox" name="groupBox_4">
               <property name="title">
                <string/>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_8">
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout_7">
                  <property name="sizeConstraint">
                   <enum>QLayout::SizeConstraint::SetDefaultConstraint</enum>
                  </property>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="startButton">
               <property name="font">
                <font>
                 <pointsize>12</pointsize>
                 <bold>true</bold>
                </font>
               </property>
               <property name="focusPolicy">
                <enum>Qt::FocusPolicy::StrongFocus</enum>
               </property>
               <property name="styleSheet">
                <string notr="true">:enabled {background-color: rgb(85, 170, 127); color: rgb(255, 255, 255);}
:disabled {background-color: rgb(209, 209, 209); color: rgb(255, 255, 255);}
</string>
               </property>
               <property name="text">
                <string>START GAME</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1100</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuGame">
    <property name="title">
     <string>Game</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="separator"/>
    <addaction name="actionHighScores"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHelpKeys"/>
    <addaction name="separator"/>
    <addaction name="actionCheckForUpdate"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuGame"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionHelpKeys">
   <property name="text">
    <string>Keys</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
   <property name="shortcut">
    <string>Shift+F1</string>
   </property>
  </action>
  <action name="actionCheckForUpdate">
   <property name="text">
    <string>Check for Update</string>
   </property>
  </action>
  <action name="actionHighScores">
   <property name="text">
    <string>Top scores</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="nextblock.qrc"/>
 </resources>
 <connections/>
</ui>
